added J in Ut

chakrabortydeepro opened the pull request.

wujastyk commented:
I have added MS J to the directory of files at Saktumiva.  I have renamed the files of the uttaratantra, so you should do a git pull to update your local files.

chakrabortydeepro commented:
Alright, Prof. Wujastyk.

On Fri, Apr 5, 2024 at 11:06 PM Dominik Wujastyk ***@***.***>
wrote:

> I have added MS J to the directory of files at Saktumiva. I have renamed
> the files of the uttaratantra, so you should do a git pull to update your
> local files.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wujastyk/sushrutaproject/pull/261#issuecomment-2040968533>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIIRWBIGT3NUGMMEOZSDU4LY357E7AVCNFSM6AAAAABF2CTS2CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDANBQHE3DQNJTGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Deepro Chakraborty
(he/him)
PhD candidate
Department of History, Classics, and Religion
University of Alberta

*The University of Alberta acknowledges that we are located on ᐊᒥᐢᑿᒌᐚᐢᑲᐦᐃᑲᐣ
(Amiskwacîwâskahikan) Treaty 6 territory, and respects the history,
languages, and cultures of the First Nations, Métis, Inuit, and all First
Peoples of Canada, whose presence continues to enrich our institution.*
